insights/parsers/satellite_installer_configurations.py

```diff
@@ -27,7 +29,13 @@ class CustomHiera(YAMLParser):
         582
     """
     def parse_content(self, content):
+        self.is_invalid_yaml = False
         try:
             super(CustomHiera, self).parse_content(content)
         except SkipException:
             pass
+        except ParseException as ex:
```

Comment:
@Rohan21Lobo in the case where a parser is looking for bad or negative results, we recommend writing a second "negative" or "dirty" parser that will detect the bad condition and not do both in the same parser.  See the documentation on [exceptions](https://insights-core.readthedocs.io/en/latest/exception_model.html#case-3b-parsing-data-to-find-errors-dirty-parser).  The purpose of `SkipException` is to remove the parser from the dependency tree, and here its being circumvented.  This means that any rule that is using this parser cannot rely on the dependency checking to control its execution and it will have to check the `is_invalid_yaml` flag.  Having a negative parser would allow rules that are explicitly looking for either the positive or the negative case to simply rely on the dependency checking to execute.